.github/workflows/assets/swagger.yaml

```diff
@@ -2064,6 +2066,76 @@ paths:
               example:
                 Code: INTERNAL_SERVER_ERROR
                 Message: Refund failed due to an unexpected error
+  /v2.21/getRates:
```

Comment:
I don't want to add these to `ext-documentation` repo yet (not sure I want merchants aware of this).

When we do the story to use the file from that repo to generate the code, we will need to pull this out and then merge it with that file before generating.

Reply:
I built these endpoints off:

https://docs.withreach.com/reference/getrates-1
https://docs.withreach.com/reference/getrate